schema.graphql

```diff
@@ -19,6 +19,10 @@ input AddFileAndMetadataInput {
 }
 
 input AddFileMetadataWithFileIdInput {
+  metadataInputValues: [AddFileMetadataWithFileIdInputValues!]!
+}
+
+input AddFileMetadataWithFileIdInputValues {
```

Comment:
Maybe rename this `ÀddFileMetadataInputValues` the "FileId" seems a bit superfluous.

Reply:
I've updated the case class which will update this input as well. I think this name change will break the API update so we'll have to think about how to deploy this.